README.md

```diff
@@ -1,4 +1,4 @@
-This is the official repository for the [WebPagetest](http://www.webpagetest.org/) web performance testing code.
+This is a fork of the official repository for the [WebPagetest](http://www.webpagetest.org/) web performance testing code.
```

Comment:
No, so don't. I closed the pull request. I actually meant to merge WPO-Foundation/webpagetest into my own fork, got muddled with Github's GUI and actually requested it the opposite.  That'll teach me to try and do stuff when I don't have access to a CLI. Sorry for wasting your time.  If you can delete all this, go ahead @pmeenan !